pennylane/measurements/mid_measure.py

```diff
@@ -24,8 +24,8 @@
 from .measurements import MeasurementProcess, MidMeasure
 
 
-def measure(wires):  # TODO: Change name to mid_measure
-    """Perform a mid-circuit measurement in the computational basis on the
+def measure(wires: Wires, reset: Optional[bool] = False):  # TODO: Change name to mid_measure
```

Comment:
Is this TODO ever going to happen?

Reply:
Lol. I can add it to this PR if needed

Reply:
I don't think we need to change to `mid_measure` :+1:

Reply:
In that case I'll remove the `TODO`